letsencrypt/tests/cli_test.py

```diff
@@ -93,6 +94,29 @@ def test_help(self):
             from letsencrypt import cli
             self.assertTrue(cli.USAGE in out)
 
+    def test_configurator_selection(self):
+        real_plugins = disco.PluginsRegistry.find_all()
+        args = ['--agree-eula', '--apache', '--authenticator', 'standalone']
+
+        with mock.patch('letsencrypt.cli.plugins_testable') as plugins:
```

Comment:
Just do `mock.patch('letsencrypt.cli.plugins_disco.PluginsRegistry')`, otherwise `find_all` is called twice during the binary call.